Docs/Generali/piano_di_qualifica/03_qual_di_prod.tex

```diff
@@ -0,0 +1,72 @@
+\section{Qualità di prodotto}
+Per garantire la qualità del software, il gruppo adotta il modello
+definito dallo standard ISO/IEC 9126. Il quadro di riferimento 
+delle metriche selezionate è definito nelle \textit{Norme di Progetto}.
+\subsection{Documenti}
+\input{template/prod_quality_table.tex}
+
+\prodQualityTable{
+    Comprensibilità &
+     Capacità di un documento di essere letto e compreso facilmente &
+     MPD1\tabularnewline
+    Correttezza &
+     Proprietà di un documento privo di errori ortografici e sintattici &
+     MPD2\tabularnewline
+
+    \captionline\caption{Obiettivi di qualità per i documenti}
+
+}
+
+\subsection{Software}
+
+\prodQualityTable{
+
+    Funzionalità & 
+     Capacità di un prodotto software di fornire le funzioni, espresse ed implicite, necessarie 
+     per operare in un determinato contesto & 
+     MPD3, MPD4, MPD5\tabularnewline
+    Affidabilità & 
+     Capacità di un prodotto software di mantenere il livello di prestazione 
+     quando viene utilizzato in condizioni specificate &
+     MPD6, MPD7, MPD8\tabularnewline
+    Usabilità & 
+     Capacità di un prodotto software di essere compreso e appreso & 
+     MPD9\tabularnewline
+    Efficienza & 
+     Capacità di un prodotto software di realizzare le funzioni richieste nel 
+     minor tempo possibile ed utilizzando nel miglior modo le risorse necessarie &
```

Comment:
aggiungi: solo le risorse strettamente necessarie

Reply:
https://cdn.standards.iteh.ai/samples/35755/21d54ab4340b47acb715a549116fbc33/ISO-IEC-25030-2007.pdf
(definizioni riprese dallo standard 9126)
5.4 Software quality model
The quality of a system is the result of the quality of the system elements and their interaction. This
International Standard focuses on the quality of the software as part of a system.
Software quality is the capability of the software product to satisfy stated and implied needs when
used under specified conditions. The software product quality model provided in ISO/IEC 9126-1
[ISO/IEC 25010] defines six quality characteristics:
• Functionality: The capability of the software product to provide functions which meet stated
and implied needs when the software is used under specified conditions.
• Reliability: The capability of the software product to maintain a specified level of performance when used under specified conditions
• Usability: The capability of the software product to be understood, learned, used and attractive to the user, when used under specified conditions.
• Efficiency: The capability of the software product to provide appropriate performance, relative to the amount of resources used, under stated conditions.
• Maintainability: The capability of the software product to be modified. Modifications may
include corrections, improvements or adaptation of the software to changes in environment,
and in requirements and functional specifications.
• Portability: The capability of the software product to be transferred from one environment to
another.